.sync.yml

```diff
@@ -1,3 +1,7 @@
 ---
 appveyor.yml:
   delete: true
+.travis.yml:
+  extras:
+  - rvm: 2.1.9
+    script: bundle exec rake rubocop
```

Comment:
Would it be possible to remove this line break.

Reply:
not sure

Reply:
Matches the style of all other .sync.yml files I've checked

Reply:
Understand now, have resolved

Reply:
Have you pushed this change? I can still see it.

Reply:
Done

Reply:
Perfect